src/socket.js

```diff
@@ -70,7 +70,14 @@ export default class SocketClient extends Module {
 		this.on(':command:reconnect', this.reconnect, this);
 
 		this.on(':command:do_authorize', challenge => {
-			this.log.warn('Unimplemented: do_authorize', challenge);
+			// this.log.warn('Unimplemented: do_authorize', challenge);
+			// We don't have our own IRC connection yet, so the site's chat has to do.
```

Comment:
Not a big thing, but if we can getUser() and it has an oauth token we should just open a quick websocket, send three messages, and close it without waiting for a response. It's not like IRC is hard if you just need to send a message.

I can do that though.